portal-impl/src/com/liferay/portlet/internal/PortletURLImpl.java

```diff
@@ -426,6 +426,17 @@ public void setBeanParameter(PortletSerializable portletSerializable) {
 
 		if (Validator.isNull(paramName)) {
 			paramName = portletSerializableClass.getSimpleName();
+
+			// If RenderURL.setBeanParameter(PortletSerializable) is called from
+			// within a bean portlet that uses Spring for IoC, and if Spring
+			// dynamically created the bean at runting using CGLib, then the
+			// parameter name will be prefixed by "$$EnhanderBySpringCGLib"
+
+			int pos = paramName.indexOf("$$EnhancerBySpringCGLIB");
```

Comment:
@ngriffin7a what do I do to trigger this?

Reply:
@shuyangzhou It would get triggered if you called `renderURL.setBeanParameter(fooBean);` -- if you need a separate demo, I can get you one. Please let me know.

Reply:
Is there an UI way I can trigger it with your demo portlets?